writing-specifications.rst

```diff
@@ -261,26 +278,18 @@ In the following examples, ``Die Hard``, ``gauge-java``, and ``100`` are values
    * Create a “gauge-java” project
    * Write “100” line specification
 
-**Note:** Renaming the parameter will not rename the usages inside the
-method. By design, the renamed parameter is considered a new parameter.
-Therefore the usage of the old parameter (if any) has to be fixed
-manually to resolve the corresponding compilation issue.
-
-2. Dynamic Parameters
+Dynamic Parameters
 ^^^^^^^^^^^^^^^^^^^^^
 
 Dynamic parameters are used as placeholders instead of actual values. 
 These parameters are used when referring to a table column value of a data table. Data tables are defined at the beginning of a spec. 
 Dynamic parameters are also used as values in a Concept.
 Dynamic parameters have the following syntax: ``<dynamic_param>``.
 
-Dynamic parameters are primarily used when referring to a table column value in :ref:`table_driven_execution`,
-or while passing values to :ref:`concept`.
-
 Example: Dynamic Parameters used in a spec which has a data table
 ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
 In the following example, ``<name>`` is a dynamic parameter. 
-``<name>`` can take the values mentioned in the "name" column of the table. In this case, "Vishnu", "Prateek", or "Navaneeth".
+``<name>`` can take the values mentioned in the "name" column of the table. In this case, the values are "Vishnu", "Prateek", or "Navaneeth".
```

Comment:
Please change the names to `Alice`, `Bob` and `Eve` Please make sure that all occurrences of Prateek, Vishnu and Navaneeth are changed to these values.